packages/@react-spectrum/s2/stories/Table.stories.tsx

```diff
@@ -10,49 +10,556 @@
+    onResizeStart: action('onResizeStart'),
+    onResizeEnd: action('onResizeEnd'),
+    columns: resizeColumn,
+    isSortable: false
+  },
+  name: 'resizing only table'
+};
+
+export const ResizingSortableTable = {
+  render: SortableResizableTable,
+  args: {
+    onResize: action('onResize'),
+    onResizeStart: action('onResizeStart'),
+    onResizeEnd: action('onResizeEnd'),
+    columns: sortResizeColumns,
+    isSortable: true
+  },
+  name: 'resizing and sortable table'
+};
+
+function AsyncLoadingExample(props) {
+  interface Item {
+    data: {
+      id: string,
+      url: string,
+      title: string
+    }
+  }
+
+  let columns = [
+    {name: 'Score', id: 'score', defaultWidth: 100, allowsResizing: true, allowsSorting: true},
+    {name: 'Title', id: 'title', allowsResizing: true, allowsSorting: true, isRowHeader: true},
+    {name: 'Author', id: 'author', defaultWidth: 200, allowsResizing: true, allowsSorting: true},
+    {name: 'Comments', id: 'num_comments', defaultWidth: 100, allowsResizing: true, allowsSorting: true}
+  ];
+
+  let list = useAsyncList<Item>({
+    getKey: (item) => item.data.id,
+    async load({signal, cursor}) {
+      let url = new URL('https://www.reddit.com/r/upliftingnews.json');
+      if (cursor) {
+        url.searchParams.append('after', cursor);
+      }
+      let res = await fetch(url.toString(), {signal});
+      let json = await res.json();
+      return {items: json.data.children, cursor: json.data.after};
+    },
+    sort({items, sortDescriptor}) {
+      return {
+        items: items.length > 0 ? items.slice().sort((a, b) => {
+          if (sortDescriptor.column != null) {
+            let cmp = a.data[sortDescriptor.column] < b.data[sortDescriptor.column] ? -1 : 1;
+            if (sortDescriptor.direction === 'descending') {
+              cmp *= -1;
+            }
+            return cmp;
+          } else {
+            return 1;
+          }
+        }) : []
+      };
+    }
+  });
+
+  return (
+    <div>
+      <ActionButton styles={style({marginBottom: 8})} onPress={() => list.remove(list.items[0].data.id)}>Remove first item</ActionButton>
+      <Table {...props} aria-label="Reddit table" sortDescriptor={list.sortDescriptor} onSortChange={list.sort} selectedKeys={list.selectedKeys} onSelectionChange={list.setSelectedKeys} loadingState={list.loadingState} onLoadMore={list.loadMore} styles={style({width: '[1000px]', height: '[400px]'})}>
+        <TableHeader columns={columns}>
+          {(column) => (
+            <Column {...column}>
+              {column.name}
+            </Column>
+          )}
+        </TableHeader>
+        <TableBody items={list.items}>
+          {item =>
+            (<Row id={item.data.id} columns={columns}>
+              {(column) => {
+                return column.id === 'title'
+                  ? <Cell textValue={item.data.title}><Link isQuiet><a href={item.data.url} target="_blank">{item.data.title}</a></Link></Cell>
```

Comment:
It also looks like there is an extra focus stop in the Link cells?

Reply:
good catch, forgot to update the Link implementation to match what we have in S2 now so had the extra `<a>`